release_docs/RELEASE.txt

```diff
@@ -409,7 +409,15 @@ Bug Fixes since HDF5-1.12.2 release
 
     Java Library
     ------------
-    -
+    - Fixed switch case 'L' block missing a break statement.
+
+      The HDF5Array.arrayify method is missing a break statement in the case 'L': section
```

Comment:
This should probably be "was missing" and "caused" to agree with "Fixed" and "was fixed".